packages/xstate-react/test/createActorContext.test.tsx

```diff
@@ -470,4 +470,40 @@ describe('createActorContext', () => {
 
     expect(screen.getByTestId('value').textContent).toBe('84');
   });
+
+  it('should merge createActorContext options with options passed to the provider', () => {
+    const events = [];
+    const SomeContext = createActorContext(
+      createMachine({
+        types: {
+          context: {} as { count: number },
+          input: {} as number
+        },
+        context: ({ input }) => ({ count: input })
+      }),
+      {
+        inspect: (ev) => {
+          events.push(ev);
+        }
+      }
+    );
+
+    const Component = () => {
+      const count = SomeContext.useSelector((state) => state.context.count);
+
+      return <div data-testid="value">{count}</div>;
+    };
+
+    const App = () => {
+      return (
+        <SomeContext.Provider options={{ input: 10 }}>
```

Comment:
This test doesn't check that those `options` here were used - if the test is meant to check the merging behavior then we should have assertions validating that both of those are preserved somehow.

Reply:
Improved the test to check that `count: 10`

Reply:
It's already testing the top-level option by reading the `events` array